src/mpid/ch4/shm/src/shm_types.h

```diff
@@ -17,6 +17,9 @@ typedef enum {
     MPIDI_SHM_XPMEM_SEND_LMT_REQ,       /* issued by sender to initialize a LMT;
                                          * receiver transfers data in callback.  */
     MPIDI_SHM_XPMEM_SEND_LMT_ACK,       /* issued by receiver to notify completion of LMT */
+    MPIDI_SHM_XPMEM_SEND_LMT_RTS,
+    MPIDI_SHM_XPMEM_SEND_LMT_CTS,
+    MPIDI_SHM_XPMEM_RECV_LMT_ACK,
 #endif
```

Comment:
Can you add a comment to simply describe each packet type?

Reply:
Please check the new added comments.